canonical-websites/www.ubuntu.com
# Updated global nav to 1.14

## Done

- Updated global nav to 1.14

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check that the global nav no longer points to mongoose.u.c

## Issue / Card

Fixes #4270
